pkgs/applications/misc/synergy/default.nix

```diff
@@ -50,7 +50,7 @@ stdenv.mkDerivation rec {
   patches = [
     # Without this OpenSSL from nixpkgs is not detected
     ./darwin-non-static-openssl.patch
-  ] ++ lib.optionals (stdenv.isDarwin && !(darwin.apple_sdk.frameworks ? UserNotifications)) [
+  ] ++ lib.optionals (stdenv.isDarwin && isNull UserNotifications) [
```

Comment:
```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
```

Reply:
It's even better if the line can be removed.

Reply:
Done in commit 7bf7cb5a576fc72ccca5bb217fbb625e0c1a27af.